magento-research/pwa-studio
# Peregrine REST API Hook

## Description
This PR introduces a hook into Peregrine for using Magento's REST API.

## Related Issue





Closes #1295 .

## Verification Steps


There aren't any components that use this REST API hook yet.
Verify that the unit tests are written correctly.

## How have YOU tested this?




`yarn test`

## Screenshots / Screen Captures (if appropriate):

## Proposed Labels for Change Type/Package




## Checklist:
